projects/frontend/data-pipelines/gui/e2e/fixtures/e2e-cy-dp-failing.job.json

```diff
@@ -0,0 +1,19 @@
+{
+  "job_name": "e2e-cp-$env-placeholder$-failing-job",
+  "description": "Testing failing job, that is used in data-pipelines CI/CD for UI e2e tests. It is intended to be always available in order to speed up deployment and if deleted will be automatically re-created by the e2e tests.",
+  "config": {
+    "db_default_type": "IMPALA",
+    "contacts": {
+      "notified_on_job_failure_user_error": [],
+      "notified_on_job_failure_platform_error": [],
+      "notified_on_job_success": [],
+      "notified_on_job_deploy": []
+    },
+    "schedule": {
+      "schedule_cron": "0 */12 * * *"
+    },
+    "generate_keytab": true,
+    "enable_execution_notifications": true
+  },
+  "team": "supercollider"
```

Comment:
Please, use a generic team name, e.g. VDK. 

Note: I see supercollider is used in more files. Please, consider these also while refactoring this.

Reply:
Good catch, did refactor to "taurus" for consistency with other components tests config.ini team used.
Also, the zip files contained configs.ini, those I updated and fixed the imports to vdk-core